.github/workflows/release-please.yml

```
@@ -0,0 +1,24 @@
+# this workflow automatically creates release PRs when something gets merged into the main branch.
+# we're using the strategy documented here https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+  pull-requests: write
+
+name: release-please
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
```

```suggestion
      - uses: google-github-actions/release-please-action@v3.7.11
```

if v3 uses the latest v3 ver already, pls ignore this suggestion

---

pinning this to the SHA